developer/src/kmcmplib/meson_options.txt

```diff
@@ -0,0 +1 @@
+option('full_test', type: 'boolean', value: false)
```

Comment:
How does this get set to `true` on CI?

Reply:
It doesn't, yet. See my comment at top:

> This will require CI changes to run full tests, relating to https://github.com/keymanapp/keyman/issues/8361.